Informatics II/000/Task 3/Patertuck Solution/Ex00 Pseudocode.txt

```diff
@@ -0,0 +1,22 @@
+Input: An array A[1..n] with n distinct integers, where n ≥ 2. 
+Output: the second largest integer in A
+
+# maybe replace this part with two very small numbers for largest_number and second_number #
+if A[0] > A[1]
+	largest_number = A[0] and second_number = A[1]
+else 
+	largest_number = A[1] and second_number = A[0]
+#															 #
+
+for i from A[2 until end]
```

Comment:
A[2 until end], meaning: for (int i = 1; i < n; i++) ?

Reply:
(int i = 2; i < n; i++)

Reply:
but this was before I used a very small number to initialize largest_number and second_number

Reply:
will remove it for the better algorithm used for the c file too